src/cmd/team_test.go

```diff
@@ -0,0 +1,106 @@
+package cmd_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/opslevel/cli/cmd"
+	"github.com/opslevel/opslevel-go/v2024"
+)
+
+const (
+	teamFileName = "test_team.yaml"
+	teamName     = "CLI Test Team"
+)
+
+func Test_TeamCRUD(t *testing.T) {
+	// Create Team
+	teamToCreate := opslevel.TeamCreateInput{
+		Name:             teamName,
+		Responsibilities: opslevel.RefOf("all the things"),
+	}
+	teamId, err := createTeam(teamToCreate)
+	if err != nil {
+		t.Error(err)
+		t.FailNow()
+	}
+
+	// Get Team
+	createdTeam, err := getTeam(teamId)
```

Comment:
When it comes to all of these CRUD functions that the code has been divided into, we could actually pass in `t *testing.T` into them, and have those functions call `t.Fatal(err)` so that we don't have to examine return values and this function could be much shorter.

Reply:
There are problems with automatically failing in each function.

If `updateTeam(...)` fails to update the created test team, we need to the flexibility to delete the team outside of that function. Other tests down the road might make user of `updateTeam()` or even `getTeam()` and those tests would need to determine how to handle those situations. We might want to run `getTeam()` to prove a team has been deleted for instance.